### ISSUE TITLE
intrinsic fallback should be clarified for the HTML `video` element too

### ISSUE THREAD

**Issue by rdeltour** [mattgarrish/epub-revision#2490]
An audio resource can be used in an HTML `video` element. In this case, the element's `source` children can define the intrinsic audio fallback.

By only mentioning the `audio` element in the intrinsic fallback clarification, the EPUB spec confuses the element with the underlying resource: the **resource** requires a fallback, regardless of where it appears.

This can easily be fixed by replacing any reference to "[audio](https://html.spec.whatwg.org/multipage/#the-audio-element) element" by "[media element](https://html.spec.whatwg.org/multipage/media.html#media-element) (namely, [`audio`](https://html.spec.whatwg.org/multipage/media.html#the-audio-element) and [`video`](https://html.spec.whatwg.org/multipage/media.html#the-video-element))".

**Comment by iherman** [comment]
@rdeltour I tried to locate the sentence(s) that you seem to refer to, but I could not. Can you give a more exact reference?

**Comment by rdeltour** [comment]
> Can you give a more exact reference?

It's all in the clarification of [HTML `audio` fallbacks](https://www.w3.org/TR/2022/CRD-epub-33-20221124/#sec-fallbacks-audio) section.

**Comment by iherman** [comment]
I saw that one, but I do not understand the issue any more. That section is specifically on the audio element, and the note makes it clear that for a video element, because it refers to an exempt resource, fallbacks are not relevant. I am officially lost.

**Comment by rdeltour** [comment]
The issue is that in HTML `audio` and `video` elements are very similar and can both contain audio **or** video resources.

See that note in HTML [Media elements](https://html.spec.whatwg.org/#media-elements) section
> Both [audio](https://html.spec.whatwg.org/#the-audio-element) and [video](https://html.spec.whatwg.org/#the-video-element) elements can be used for both [audio](https://html.spec.whatwg.org/#the-audio-element) and [video](https://html.spec.whatwg.org/#the-video-element). The main difference between the two is simply that the audio element has no playback area for visual content (such as video or captions), whereas the video element does.

So in EPUB:
- the fact that embedded flow content is not an intrinsic fallback
- and that only `source` children provide intrinsic fallback

also applies to the `video` element, and is relevant when that `video` element represents an audio resource.

In other words, it is not because video **resources** are exempt, that we don't have to clarify the intrinsic fallback mechanism of the `video` **element**, since that element can also be used for audio.

**Comment by iherman** [comment]
Wow. I did not realize that the HTML spec is so "liberal" (although, personally, I think this is more confusing than helpful, but who am I to say that?). That was the reason for my confusion.

In other words, what you propose is to rewrite the whole 3.5.2.1. Section by referring to (HTML) media elements instead of audio elements only.

I will make a run at this if this is indeed the plan.

**Comment by rdeltour** [comment]
> In other words, what you propose is to rewrite the whole 3.5.2.1. Section by referring to (HTML) media elements instead of audio elements only.

yes, exactly 😊
